frontend/src/components/pages/teacher/DisplayAssessmentsPage.tsx

```diff
@@ -78,23 +79,26 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
 
   return (
     <>
-      <HeaderWithButton
-        buttonText="Add Assessment"
-        showButton={!!formattedData?.length}
-        targetRoute={Routes.DISTRIBUTE_ASSESSMENT_PAGE}
-        title="Assessments"
-      />
+      <VStack align="left" mb={10}>
+        <HeaderWithButton
+          buttonText="Add Assessment"
+          showButton={!!formattedData?.length}
+          targetRoute={Routes.DISTRIBUTE_ASSESSMENT_PAGE}
+          title="Assessments"
+        />
+      </VStack>
+
       {loading && (
         <Center flex="1" margin="15%">
           <LoadingState />
         </Center>
       )}
       {error && (
-        <Box height="100%" mt={10}>
+        <Box>
           <ErrorState />
         </Box>
       )}
-      {!!formattedData?.length && !loading && !error && (
+      {!!formattedData?.length && !loading && error && (
```

Comment:
```suggestion
      {!!formattedData?.length && !loading && !error && (
```
is it supposed to be this?